src/katsdpcontroller/generator.py

```diff
@@ -1357,7 +1399,7 @@ def get_baseline_index(a1, a2):
             if isinstance(stream, product_config.GpucbfBaselineCorrelationProductsStream):
                 renames = ["chan-range", "rx.synchronised", "xeng-clip-cnt"]
             elif isinstance(stream, product_config.GpucbfTiedArrayChannelisedVoltageStream):
-                renames = ["chan-range", "beng-clip-cnt"]
+                renames = ["chan-range", "delay", "quantiser-gain", "weight", "beng-clip-cnt"]
```

Comment:
It actually looks like chan-range isn't specified (in the ICD) for B streams, but I feel like that's an oversight. @james-smith-za are you happy if we add it to the ICD?

Reply:
This is going to make sensors called `{stream.name}.{i}.delay` etc, while the ICD specifies just a single sensor called `{stream_name}.delay`.

I think we'll need a new aggregate sensor class for these, which should probably take the child sensor with the latest timestamp.

Reply:
These are good suggestions.

Reply:
> This is going to make sensors called `{stream.name}.{i}.delay` etc, while the ICD specifies just a single sensor called `{stream_name}.delay`.

Indeed, I was meant to flag that on the ticket earlier when I was going through the full `?sensor-list`, must've got sidetracked. Are you fine if I defer the creation of that new aggregate sensor to the katcp control ticket? Or can I just create a separate ticket for that?

As for the chan-range sensor, I'll make a suggestion on the ICD now.

Reply:
Definitely don't do it in this PR. I've implemented on my `NGC-446-beamformer-control` branch, which builds on top of this one.

Reply:
Noted 👍